src/components/RequestInformation/RequestInformation.js

```diff
@@ -58,54 +80,54 @@ const RequestInformation = ({
-                  disabled={isEditForm}
-                  fullWidth
-                >
-                  {requestTypeOptions.map(({
-                    id,
-                    value,
-                  }) => (
-                    <FormattedMessage
-                      id={id}
-                      key={id}
-                    >
-                      {translatedLabel => (
-                        <option value={value}>
-                          {translatedLabel}
-                        </option>
-                      )}
-                    </FormattedMessage>
-                  ))}
-                </Field>
-              }
-              {singleRequestTypeVisible &&
-                <KeyValue
-                  label={<FormattedMessage id="ui-requests.requestType" />}
-                  value={
-                    <span data-test-request-type-text>
-                      <FormattedMessage id={requestTypeOptions[0].id} />
-                    </span>
-                  }
-                />
-              }
-              {isEditForm &&
+              {!requestTypeError && (isEditForm ?
                 <KeyValue
                   label={<FormattedMessage id="ui-requests.requestType" />}
                   value={<FormattedMessage id={requestTypesTranslations[request.requestType]} />}
-                />
+                /> :
+                <Field
+                  data-testid="requestTypeDropDown"
+                  name={REQUEST_FORM_FIELD_NAMES.REQUEST_TYPE}
+                  validateFields={[]}
+                  validate={validateRequestType}
+                >
+                  {({
+                    input,
+                    meta,
+                  }) => {
+                    const error = (meta.touched && meta.error) || null;
+
+                    return (
+                      <Select
+                        {...input}
+                        label={<FormattedMessage id="ui-requests.requestType" />}
+                        disabled={isRequestTypeDisabled}
+                        error={error}
+                        fullWidth
+                        required
+                      >
+                        <FormattedMessage id="ui-requests.actions.selectRequestType">
+                          {optionLabel => <option value="">{optionLabel}</option>}
+                        </FormattedMessage>
+                        {requestTypeOptions.map(({
+                          id,
+                          value,
+                        }) => (
+                          <FormattedMessage
+                            id={id}
+                            key={id}
+                          >
+                            {translatedLabel => (
+                              <option value={value}>
+                                {translatedLabel}
+                              </option>
+                            )}
+                          </FormattedMessage>
+                        ))}
```

Comment:
Why don't you use `dataOptions` prop to render options?

Reply:
it will reduce your code by almost 20 lines :)

Reply:
@mariia-aloshyna Some of the options for `Select` component are generated dynamically.
If I create a separate `dataOptions` array we will not reduce my code.

cc @Dmitriy-Litvinenko

Reply:
Furthermore, I see that our code base contains both approaches.

Reply:
I approved this PR, but still, it doesn't look like the additional 20 lines of code that are already implemented by the Select component.
`dataOptions={requestTypeOptions.map(({ id, value }) => ({ label: formatMessage({ id }), value }))}`